default/stringtables/en.txt

```diff
@@ -4146,19 +4111,46 @@ Parts: %3%
 
 '''
 
-# FIXME
-# will need dummy vals 1-8 and 12 for numbering to remain compatible with original ENC_SHIP_DESIGN_DESCRIPTION_STR
+# Detailed ship design information for a given species
+# %1% species for this design
+# %2% total damage from all weapon types
+# %3% weapon shots per round
+# %4% structure
+# %5% shield
+# %6% detection range
+# %7% stealth
+# %8% starlane speed
+# %9% fuel
+# %10% population
+# %11% troops
+# %12% fighters
+# %13% total damage per round of fighters
+# %14% launch bay capacity
+# %15% average attack strength
+# %16% average attack strength/pp
+# %17% mock enemy attack strength
+# %18% mock enemy shields
+# %19% mock average attack strength
+# %20% mock average attack strength/pp
 ENC_SHIP_DESIGN_DESCRIPTION_STATS_STR
 '''
-For species: %18%
-Total [[encyclopedia DAMAGE_TITLE]]: %17%    [[metertype METER_STRUCTURE]]: %8%    [[metertype METER_SHIELD]]: %9%
-[[metertype METER_SPEED]]: %13%    [[metertype METER_FUEL]] Capacity: %14%
-[[metertype METER_DETECTION]]: %10%    [[metertype METER_STEALTH]]: %11%
-Colonization Capacity: %15%    [[metertype METER_TROOPS]]: %16%
-Estimated combat strength (ignoring shields):  %19$.f   (%20$.2f per PP)
-Against enemy attack %21$.1f and shields %22$.1f:       %23$.f   (%24$.2f per PP)
+For [[encyclopedia ENC_SPECIES]]: %1%
+Total [[encyclopedia DAMAGE_TITLE]]: %2%    [[PC_DIRECT_WEAPON]] Shots: %3%
+[[metertype METER_STRUCTURE]]: %4%    [[metertype METER_SHIELD]]: %5%
+[[metertype METER_SPEED]]: %6%    [[metertype METER_FUEL]] [[METER_CAPACITY]]: %7%
+[[metertype METER_DETECTION]]: %8%    [[metertype METER_STEALTH]]: %9%
```

Comment:
Um, not sure it is, I've recompiled from this branch and both are swapped in my display.